src/sentry/seer/similarity/similar_issues.py

```diff
@@ -69,15 +69,17 @@ def get_similarity_data_from_seer(
         )
         return []
 
-    normalized = []
+    if not response_data:
+        return []
```

Comment:
should we log if this happens? doesn't seem like it should in theory.

Reply:
It'll be included in the metric, but this is a totally normal case - it will happen whenever Seer doesn't find any groups that are matching or even close enough to count as similar.

(You may be being thrown off by the change to dig out the `responses` data earlier, making `response_data` now the value of `response.data["responses"]`.)